packages/type/README.md

```diff
@@ -25,15 +25,250 @@ yarn add @carbon/type
 the IBM Design Language. You can use all of these features either in
 JavaScript or Sass. These features include:
 
-| Feature      | Description                                                                                                  |
-| ------------ | ------------------------------------------------------------------------------------------------------------ |
-| Font face    | Include IBM Plex™ font faces in your application. Uses Google fonts                                          |
-| Type classes | Helpers to use type styles directly. Not included by default                                                 |
-| Font family  | Defines the font stack for IBM Plex™ in your application, provides helpers for working with font definitions |
-| Reset        | Provides a high-level CSS Reset to use in your project                                                       |
-| Scale        | Provides the type scale. Can access the size at any given step                                               |
-| Spacing      | Provides a high-level spacing scale to be used when laying out type                                          |
-| Styles       | Provides type styles for your application                                                                    |
+| Feature                       | Description                                                                                                  |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------ |
+| [Font face](#font-face)       | Include IBM Plex™ font faces in your application. Uses Google fonts                                          |
+| [Type classes](#type-classes) | Helpers to use type styles directly. Not included by default                                                 |
+| [Font family](#font-family)   | Defines the font stack for IBM Plex™ in your application, provides helpers for working with font definitions |
+| [Reset](#reset)               | Provides a high-level CSS Reset to use in your project                                                       |
+| [Scale](#type-scale)          | Provides the type scale. Can access the size at any given step                                               |
+| [Styles](#type-styles)        | Provides type styles for your application                                                                    |
```

Comment:
Two questions here --
* Would it make sense to include a couple examples of what styles would be included in parentheses? (personally I got mixed up between styles and scale earlier this week)
* Can styles include scale? (But not the other way around of course)

Reply:
> * So would styles include scale? (But not the other way around of course)

Yup!